web/js/jquery.gerbera.config.js

```diff
@@ -488,7 +488,9 @@ $.widget('grb.config', {
               // already handled
             break;
             default:
-              console.log(`${item.item}: Unknown type '${item.type}'`);
+              if (('aid' in item)) {
```

Comment:
I assume this is not in here intentionally

Reply:
It is. Added comment to code to explain. And also added entries to json, so properties have help.